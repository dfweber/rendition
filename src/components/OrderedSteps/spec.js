/* globals expect, describe, it */
import React from 'react'
import renderer from 'react-test-renderer'
import noop from 'lodash/noop'
import { Provider, OrderedSteps } from '../../../dist'

describe('OrderedSteps component', () => {
  it('should match the stored snapshot', () => {
    const steps = [
      { label: 'Do this', complete: true },
      { label: 'Then this', complete: true },
      { label: 'and then this!', complete: false }
    ]
    const component = renderer.create(
      <Provider>
        <OrderedSteps
          m={3}
          activeStepIndex={1}
          steps={steps}
          onStepSelected={noop}
        />
      </Provider>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
