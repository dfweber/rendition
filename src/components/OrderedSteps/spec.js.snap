// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderedSteps component should match the stored snapshot 1`] = `
.c0 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c2 {
  box-sizing: border-box;
}

.c5 {
  margin-left: auto;
  margin-right: auto;
}

.c12 {
  margin-left: 16px;
  margin-right: 16px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c16 {
  font-weight: 700;
}

.c17 {
  margin-left: 8px;
}

.c1 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
}

.c1 h1,
.c1 h2,
.c1 h3,
.c1 h4,
.c1 h5,
.c1 h6,
.c1 button {
  font-family: CircularStd,Arial,sans-serif;
}

.c14 path {
  fill: #5b82a7;
}

.c10 {
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  opacity: 1;
  display: inline-block;
}

.c10:active,
.c10:hover {
  color: hsl(196.29999999999995,100%,37.5%);
}

.c11 {
  margin-left: 8px;
  color: #00AEEF;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 50%;
  border: none;
  color: #fff;
  background-color: #1AC135;
  height: 24px;
  width: 24px;
}

.c15 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 50%;
  border: none;
  color: #fff;
  background-color: #00AEEF;
  height: 24px;
  width: 24px;
}

.c18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 50%;
  border: solid 1px #bbc8d6;
  color: #bbc8d6;
  background-color: #fff;
  height: 24px;
  width: 24px;
}

.c4 {
  margin: 16px;
}

<div
  className="c0 c1"
>
  <div
    className="c2 c3 c4"
  >
    <div
      className="c2 c5 c6"
    >
      <div
        className="c2 c7"
      >
        <div
          className="c2 c8"
        >
          <div
            className="c2 c9"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-check fa-w-16 "
              data-icon="check"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <a
            className="c10 c11"
            color="primary.main"
            onClick={[Function]}
          >
            Do this
          </a>
        </div>
        <div
          className="c2 c12 c13"
        >
          <svg
            className="c14"
            disabled={false}
            height="9"
            viewBox="0 0 29.4 9"
            width="29.4"
            x="0"
            xmlns="http://www.w3.org/2000/svg"
            y="0"
          >
            <g
              transform="translate(1.000000, 4.000000)"
            >
              <path
                d="M20-4c0.1,0,0.2,0,0.2,0.1l0,0l7.9,4c0.3,0.1,0.4,0.5,0.2,0.7c-0.1,0.1-0.1,0.2-0.2,0.2l0,0
			l-8,3.9c-0.3,0.1-0.6,0-0.7-0.2c0-0.1-0.1-0.2-0.1-0.2l0,0l0-3.5l-3.9,0C15.2,1,15,0.8,15,0.5l0-0.1C15,0.2,15.2,0,15.5,0l0,0
			l3.9,0l0-3.5C19.4-3.7,19.6-4,20-4L20-4z M7.5-0.1l4,0c0.3,0,0.5,0.2,0.5,0.5s-0.2,0.5-0.5,0.5l0,0l-4,0C7.2,0.9,7,0.7,7,0.4
			C7,0.1,7.2-0.1,7.5-0.1L7.5-0.1z M-0.5-0.2l4,0C3.8-0.1,4,0.1,4,0.4S3.8,0.9,3.5,0.9l0,0l-4,0C-0.8,0.8-1,0.6-1,0.3
			C-1,0-0.8-0.2-0.5-0.2L-0.5-0.2z"
                fillRule="nonzero"
              />
            </g>
          </svg>
        </div>
      </div>
      <div
        className="c2 c7"
      >
        <div
          className="c2 c8"
        >
          <div
            className="c2 c15"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-check fa-w-16 "
              data-icon="check"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <div
            className="c16 c17 "
          >
            Then this
          </div>
        </div>
        <div
          className="c2 c12 c13"
        >
          <svg
            className="c14"
            disabled={false}
            height="9"
            viewBox="0 0 29.4 9"
            width="29.4"
            x="0"
            xmlns="http://www.w3.org/2000/svg"
            y="0"
          >
            <g
              transform="translate(1.000000, 4.000000)"
            >
              <path
                d="M20-4c0.1,0,0.2,0,0.2,0.1l0,0l7.9,4c0.3,0.1,0.4,0.5,0.2,0.7c-0.1,0.1-0.1,0.2-0.2,0.2l0,0
			l-8,3.9c-0.3,0.1-0.6,0-0.7-0.2c0-0.1-0.1-0.2-0.1-0.2l0,0l0-3.5l-3.9,0C15.2,1,15,0.8,15,0.5l0-0.1C15,0.2,15.2,0,15.5,0l0,0
			l3.9,0l0-3.5C19.4-3.7,19.6-4,20-4L20-4z M7.5-0.1l4,0c0.3,0,0.5,0.2,0.5,0.5s-0.2,0.5-0.5,0.5l0,0l-4,0C7.2,0.9,7,0.7,7,0.4
			C7,0.1,7.2-0.1,7.5-0.1L7.5-0.1z M-0.5-0.2l4,0C3.8-0.1,4,0.1,4,0.4S3.8,0.9,3.5,0.9l0,0l-4,0C-0.8,0.8-1,0.6-1,0.3
			C-1,0-0.8-0.2-0.5-0.2L-0.5-0.2z"
                fillRule="nonzero"
              />
            </g>
          </svg>
        </div>
      </div>
      <div
        className="c2 c7"
      >
        <div
          className="c2 c8"
        >
          <div
            className="c2 c18"
          >
            <div
              className=""
            >
              3
            </div>
          </div>
          <a
            className="c10 c11"
            color="primary.main"
            onClick={[Function]}
          >
            and then this!
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
